Makefile

```diff
@@ -7,7 +7,7 @@ export GOBIN ?= $(shell pwd)/bin
 export CGO_ENABLED=0
 NEOGO ?= $(GOBIN)/cli
 VERSION ?= $(shell git describe --tags --dirty --match "v*" --always --abbrev=8 2>/dev/null || cat VERSION 2>/dev/null || echo "develop")
-NEOGOORIGMOD = github.com/nspcc-dev/neo-go@v0.101.5-0.20230802075307-afa4530c7d73
+NEOGOORIGMOD = github.com/nspcc-dev/neo-go@v0.101.5-0.20230810175032-d425df855e32
```

Comment:
I'd wait for it to settle in the `master` (shouldn't take a lot of time).

Reply:
Fixed.